src/cache/pipe.rs

```diff
@@ -83,6 +83,9 @@ impl<'c, C> Pipe<'c, C> {
     }
 
     pub(crate) fn srem(&mut self, key: RedisKey, member: impl ToRedisArgs) {
+        if member.to_redis_args().is_empty() {
+            return;
+        };
```

Comment:
`.to_redis_args` is a pretty expensive call since it allocates a new `Vec<Vec<u8>>`. Intuitively, I'd assume that adding this check will generally hurt performance more than it helps.